.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,33 @@
+name: Benchmark
+
+on:
+  push:
+    paths:
+      - 'lib/**'
+  pull_request:
+    branches:
+      - master
```

Comment:
is `paths: ['lib/**']` going to be applied to pull_request too?

Reply:
No.

And it is my desired behavior: PR like this one (which modifies the CI config and `benchmark.js`) should trigger benchmark as well.

Reply:
what about `paths-ignore: ['test/**']`? Some PRs only update the unit test, so it shouldn't be necessary to run the benchmark. The [doc](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-ignoring-paths) mentions the workflow will not run if only matches 'test/**', it will still run if there is any change to other folder.